### PR TITLE
Remove project names

### PR DESCRIPTION
**Pull request by sebastienros**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/extensions/pull/4532)

### CODE REVIEW

File: src/Libraries/Microsoft.Extensions.Diagnostics.ExtraAbstractions/Http/TelemetryConstants.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public static class TelemetryConstants
     /// <summary>
     /// Request metadata key that is used when storing request metadata object.
     /// </summary>
-    public const string RequestMetadataKey = "R9-RequestMetadata";
+    public const string RequestMetadataKey = "Extensions-RequestMetadata";
```
Comment (joperezr) [inline src/Libraries/Microsoft.Extensions.Diagnostics.ExtraAbstractions/Http/TelemetryConstants.cs:14]: I'm assuming we do want to make this change, yet we do expect this to cause breaks for teams currently depending on this. @geeknoid can you confirm?

**Comment by sebastienros** [comment]
I am assuming that some of these suggestions are not good, but I can then add comments to clarify the reasons for later.

File: src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.Common/Metric.cs [hunk 2]
```diff
@@ -12,10 +12,10 @@ namespace Microsoft.Extensions.Diagnostics.HealthChecks;
 
 internal static partial class Metric
 {
-    [Counter("healthy", "status", Name = @"R9\\HealthCheck\\Report")]
```
Comment (joperezr) [inline src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.Common/Metric.cs:15]: I know that @xakep139 is also making changes around this on PR https://github.com/dotnet/extensions/pull/4482, so let's just make sure that these changes won't conflict with his.

Comment (sebastienros) [reply src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.Common/Metric.cs:15]: I see, reverting this one then

**Comment by joperezr** [comment]
Changes look good to me, but it would be good to also get a review from @xakep139 on this especially due to some of the telemetry changes being done here.

**Review by xakep139 — APPROVED** [review]

**Review by RussKie — APPROVED** [review]

File: src/Libraries/Microsoft.Extensions.Diagnostics.ExtraAbstractions/Http/TelemetryConstants.cs (hunk 1 above)
Comment (geeknoid) [reply src/Libraries/Microsoft.Extensions.Diagnostics.ExtraAbstractions/Http/TelemetryConstants.cs:14]: This should be fine, it's something teams will have to deal with when upgrading.

**Review by geeknoid — APPROVED** [review]

**Pull request merged**